src/playerhud.py

```diff
@@ -15,34 +14,34 @@ class PlayerHud:
     hud_x: int
     hud_y: int
     player_hud_avatar: arcade.Sprite
-    
 
-    def __init__(self, player: Player, hud_x: int, hud_y: int, player_hud_avatar: arcade.Sprite):
+    def __init__(
+        self, player: Player, hud_x: int, hud_y: int, player_hud_avatar: arcade.Sprite
+    ):
         self.player = player
         self.hud_x = hud_x
         self.hud_y = hud_y
-        
-        self.background_sprite = arcade.SpriteSolidColor(self.width, self.height, self.background_color)
+
+        self.background_sprite = arcade.SpriteSolidColor(
+            self.width, self.height, self.background_color
+        )
         self.background_sprite.center_x = hud_x
         self.background_sprite.center_y = hud_y
-        
+
         self.player_hud_avatar = player_hud_avatar
         self.player_hud_avatar.center_x = hud_x
         self.player_hud_avatar.center_y = hud_y - 30
-       
-        self.health_sprite = arcade.SpriteSolidColor(self.width, self.height, self.full_color)
+
+        self.health_sprite = arcade.SpriteSolidColor(
+            self.width, self.height, self.full_color
+        )
         self.health_sprite.center_x = hud_x
         self.health_sprite.center_y = hud_y
-        
+
     def update(self):
 
-        #changes healthbar length based on current ratio and left sets healthbar
+        # changes healthbar length based on current ratio and left sets healthbar
```

Comment:
Looks like this file was not formatted in the initial PR that submitted it, so your editor ended up reformatting it.  Noting because we can check formatting for Liam's next PR to make sure the formatting is working on his end.